config/locales/idv/en.yml

```diff
@@ -50,6 +50,7 @@ en:
         heading: We could not match this phone number to other records
         jobfail: Something went wrong and we cannot process your request at this time.
           Please try again.
+        timeout: Our request to verify your information timed out. Please try again.
```

Comment:
in hindsight "timed out" might be a bit jargony/maybe we can find a plain language alternative, but we can follow up with copy later